ui/lib/pki/addon/components/pki-import-pem-bundle.ts

```diff
@@ -43,15 +44,30 @@ export default class PkiImportPemBundle extends Component<Args> {
   @service declare readonly flashMessages: FlashMessageService;
 
   @tracked errorBanner = '';
+  @tracked afterImport = false;
```

Comment:
I might have missed this - where is `afterImport` being set/used in the component? 🤔

Reply:
Good catch, it's not needed!